holidays/countries/kenya.py

```diff
@@ -26,29 +26,58 @@ class Kenya(HolidayBase):
     """
 
     country = "KE"
+    special_holidays = {
+        2020: ((FEB, 11, "President Moi Celebration of Life Day	"),),
```

Comment:
Remove extra space here:
```suggestion
        2020: ((FEB, 11, "President Moi Celebration of Life Day"),),
```